### ISSUE TITLE
kwik.cx non cdn links

### ISSUE THREAD

**Issue by Modder4869** [ngomile/anigrab#8]
**Is your feature request related to a problem? Please describe.**
my internet is not stable and each time it get restarted i get new ip so all download links became unusable 

**Describe the solution you'd like**
add option to get non cdn links when downloading from animepahe

**Comment by newaccforhc** [comment]
Yes.. An option to use non-CDN links would be better for dynamic IP...

**Comment by hnumair** [comment]
Didn't want to create an issue but since we're on topic, I've been trying since yesterday but not able to bypass captcha on animepahe. It worked for the first download but after that, it's just stuck on Bypassing captcha without printing any errors.

**Comment by Modder4869** [comment]
> Didn't want to create an issue but since we're on topic, I've been trying since yesterday but not able to bypass captcha on animepahe. It worked for the first download but after that, it's just stuck on Bypassing captcha without printing any errors.

make sure you installed latest version and usually deleting the cache in `node_modules\anigrab\node_modules\flat-cache\.cache\store` usually fixes it for me when it doesn't work so dunno

**Comment by ngomile** [comment]
@Modder4869 the option to use CDN or not can be in the config file I guess and clearing cache can be a command line option. I'll see what I can do when I make the time.

**Comment by newaccforhc** [comment]
There is one more suggestion I have:
1. Ability to extract all links of an episode

Need this because:
Anime Ishuzoku Reviewers (https://pahe.win/a/3653) has 2 sources for each quality... One of them is censored while other is uncensored... So, it'll be great if we can just extract all the links and then the users can just download whichever they want using any external downloader (I'm talking about using `-u` and `-w` option)...
I'm not insisting you to add an option to select source in the tool itself because, there are only a few anime which have such dual source issues (and, I think you'll need to make many changes to do that).. Just add that feature in `-u` and `-w` flags or make a new flag ( `-all` maybe?)altogether and it'll be good enough..
Thank you!

PS: I can give you few other animes like this if you want..

**Comment by ngomile** [comment]
@newaccforhc will add config option to select fansub and fallbackFansub option.

**Comment by newaccforhc** [comment]
Okay.. Thanks!
